sdks/java/core/src/main/java/org/apache/beam/sdk/testing/CoderProperties.java

```diff
@@ -230,26 +249,62 @@
         hasItem(encodingId));
   }
 
+  /**
+   * Verifies that for the given {@code Coder<T>} and values of
+   * type {@code T}, both the structural values are equal and the encoded
+   * bytes are equal, in any {@code Coder.Context}.
+   */
   public static <T> void structuralValueConsistentWithEquals(
       Coder<T> coder, T value1, T value2)
       throws Exception {
-
     for (Coder.Context context : ALL_CONTEXTS) {
       CoderProperties.<T>structuralValueConsistentWithEqualsInContext(
           coder, context, value1, value2);
     }
   }
 
+  /**
+   * Verifies that for the given {@code Coder<T>}, {@code Coder.Context}, and
+   * values of type {@code T}, both the structural values are equal and the
```

Comment:
In this case I think it verifies that `the structural values are equal if and only if the encoded bytes are equal`